### ISSUE TITLE
Publish the sdk in a public maven repository

### ISSUE THREAD

**Issue by zaza** [cdevents/sdk-java#8]
I'm aware the sdk is in its early stages, but currently the only way to consume it requires cloning the git repository and installing it in a local maven repository. Once publicly available it would be easier to access and play with it.

**Comment by aalmiray** [comment]
FWIW the following POM elements must be defined before publishing to Maven Central:

```
$ mvn org.kordamp.maven:pomchecker-maven-plugin:1.2.0:check-maven-central
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------------< dev.cdevents:cdevents-sdk-java >-------------------
[INFO] Building cdevents-sdk-java 0.0.1
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- pomchecker-maven-plugin:1.2.0:check-maven-central (default-cli) @ cdevents-sdk-java ---
[WARNING] <url> is not defined in POM. Will use computed value from parent: 
	null
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.281 s
[INFO] Finished at: 2022-08-02T20:17:45+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.kordamp.maven:pomchecker-maven-plugin:1.2.0:check-maven-central (default-cli) on project cdevents-sdk-java: MavenCentral check failed: 
[ERROR] The POM file
[ERROR] /private/tmp/sdk-java/pom.xml
[ERROR] cannot be uploaded to Maven Central due to the following reasons:
[ERROR]  * <url> can not be blank.
[ERROR]  * <licenses> block is required but was left empty.
[ERROR]  * <developers> block is required but was left empty.
[ERROR]  * The <scm> block is required.
```

Additionally, `-sources` and `-javadoc` are needed as well. PGP signatures are a must. I can provide a PR that covers these requirements.

**Comment by m-linner-ericsson** [comment]
@aalmiray If you have the time to provide a PR, please do so. We want more people aboard 😄

**Comment by aalmiray** [comment]
This may be closed now that #9 has been merged 🎉

**Comment by tomek-pv** [comment]
Is it really done? Has the SDK been published to Maven central? If I understand what the PR does correctly, it _can_ be published now, but it hasn't happened yet, am I right? "There aren’t any releases [here](https://github.com/cdevents/sdk-java/releases)"

**Comment by aalmiray** [comment]
A Github release is not the same as publication to Maven Central. `v1.0.0-draft6` is available from Maven Central. https://search.maven.org/artifact/dev.cdevents.sdk-java/cdevents-sdk-java/v0.1.0-draft6/jar

The current configuration does not create GitHub releases. I've raised #22 to discuss options.

**Comment by afrittoli** [comment]
@tomek-pv that's a good question, I did publish what's currently tagged as [v0.1.0-draft6](https://github.com/cdevents/sdk-java/releases/tag/v0.1.0-draft6) to [maven central](https://central.sonatype.com/namespace/dev.cdevents.sdk-java) but we have no GitHub release yet - mainly because I would like the SDK Java team to check everything is in shape for that :)

**Comment by afrittoli** [comment]
@aalmiray I saw your reply after I posted mine 😅

**Comment by tomek-pv** [comment]
Thanks guys, that makes sense. Let's keep this closed and move the discussion to #22
